fhir-database-utils/src/test/java/com/ibm/fhir/database/utils/query/PredicateParserTest.java

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
I ran the tests ...  it's about 10% test coverage. 
I'd  be curious  to learn what it is when running the  IT tests

Reply:
I'm leaning on the persistence tests to provide the bulk of the coverage. We could certainly beef up the standalone tests.

Reply:
#2319